### ISSUE TITLE
Service provider class not found by Composer

### ISSUE THREAD

**Issue by connorgurney** [connorgurney/design-system#4]
Upon running `composer require gxrneyme/design-system` within a project, the following error message is received:

````
In ProviderRepository.php line 208:

  Class 'GxrneyMe\DesignSystem\DesignSystemServiceProvider' not found
````

This is because the service provider is at `src/ServiceProvider.php`, not `src/
DesignSystemServiceProvider.php` as it should be.

**Comment by connorgurney** [comment]
Fixed by eb53dc3f8009299087428e3d8fc27630774d5e3e.

**Comment by connorgurney** [comment]
Reopening this, as the issue continues. Appears to be an issue to do with the casing of the namespace.

**Comment by connorgurney** [comment]
It was actually to do with our lack of a PSR-4 autoloader within `composer.json`. Fixed in 2fe5f27500daa15b90f205693978a57125bb3b6b.
